ardupilot_gz_bringup/launch/robots/iris.launch.py

```diff
@@ -148,10 +158,52 @@ def generate_launch_description():
         output="screen",
     )
 
+    # Transform - use if the model includes "gz::sim::systems::PosePublisher"
+    #             and a filter is required.
+    # topic_tools_tf = Node(
+    #     package="topic_tools",
+    #     executable="transform",
+    #     arguments=[
+    #         "/gz/tf",
+    #         "/tf",
+    #         "tf2_msgs/msg/TFMessage",
+    #         "tf2_msgs.msg.TFMessage(transforms=[x for x in m.transforms if x.header.frame_id == 'odom'])",
+    #         "--import",
+    #         "tf2_msgs",
+    #         "geometry_msgs",
+    #     ],
+    #     output="screen",
+    #     respawn=True,
+    # )
+
+    # Relay - use instead of transform when Gazebo is only publishing odom -> base_link
+    topic_tools_tf = Node(
+        package="topic_tools",
+        executable="relay",
+        arguments=[
+            "/gz/tf",
+            "/tf",
+        ],
+        output="screen",
+        respawn=False,
+        condition=IfCondition(LaunchConfiguration("use_gz_tf")),
+    )
+
     return LaunchDescription(
         [
+            DeclareLaunchArgument(
+                "use_gz_tf", default_value="true", description="Use Gazebo TF."
```

Comment:
I'm a bit worried leaving this as default might confuse users, since they are expecting odometry from cartographer. Either way these changes look good to me

Reply:
I left it this way because the standalone examples in the README for the iris runway and iris maze do not invoke cartographer (which is part of the downstream package ardupilot_ros, where we need to update the README).